mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogEvents.cs

```diff
@@ -269,6 +294,34 @@ internal override void Accept (LogEventVisitor visitor)
 		}
 	}
 
+	public sealed class HeapRootRegisterEvent : LogEvent {
+
+		public long RootPointer { get; internal set; }
+
+		public long RootSize { get; internal set; }
+
+		public LogHeapRootSource Source { get; internal set; }
+
+		public long Key { get; internal set; }
+
+		public string Name { get; internal set; }
+
+		internal override void Accept (LogEventVisitor visitor)
+		{
+			visitor.Visit (this);
+		}
+	}
+
+	public sealed class HeapRootUnregisterEvent : LogEvent {
+
+		public long StartPointer { get; internal set; }
```

Comment:
Maybe rename this also to `RootPointer` to match new HeapRootRegisterEvent naming